### PR TITLE
Task/api 1402 queen elizabeth migration to java 12

### PR DESCRIPTION
**Pull request by jtorano-va**
Related Story: https://vasdvp.atlassian.net/browse/API-1402

### CODE REVIEW

**Comment by bschofield-va** [comment]
What's going on with https://github.com/department-of-veterans-affairs/health-apis-eligibility-and-enrollment-client/pull/23

**Comment by jblefkove-va** [comment]
Whoops. I made a separate branch to work this as a prereq for upgrading community-care-eligibility to Java 12. Let's resolve this first and I will reconcile my branch afterward.

File: ee-artifacts/pom.xml [hunk 1]
```diff
@@ -4,29 +4,41 @@
   <parent>
     <groupId>gov.va.api.health</groupId>
     <artifactId>health-apis-parent</artifactId>
-    <version>2.0.11</version>
+    <version>3.0.10</version>
```
Comment (jblefkove-va) [inline ee-artifacts/pom.xml]: 3.0.11 is available

File: ee-artifacts/pom.xml [hunk 2]
```diff
@@ -4,29 +4,41 @@
...
     <relativePath/>
   </parent>
   <artifactId>ee-artifacts</artifactId>
   <version>1.1.49-SNAPSHOT</version>
```
Comment (jblefkove-va) [inline ee-artifacts/pom.xml]: Version should go up to 2.0.0-SNAPSHOT since this is a major upgrade.

File: hand-of-queen-elizabeth/pom.xml [hunk 3]
```diff
@@ -4,25 +4,34 @@
...
-    <version>2.0.11</version>
+    <version>3.0.10</version>
     <relativePath/>
   </parent>
   <artifactId>hand-of-queen-elizabeth</artifactId>
   <version>1.1.49-SNAPSHOT</version>
   <packaging>jar</packaging>
   <properties>
     <jacoco.coverage>0.80</jacoco.coverage>
+    <service-auto-config.version>2.0.0</service-auto-config.version>
```
Comment (jblefkove-va) [inline hand-of-queen-elizabeth/pom.xml]: This can go to 3.0.11 as well

**Pull request merged**